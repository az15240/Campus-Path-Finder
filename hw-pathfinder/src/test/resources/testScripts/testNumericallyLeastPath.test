# Test the numerically sorted output

CreateGraph A
AddNode A n1
AddNode A n2
AddNode A n3
AddEdge A n1 n3 999
AddEdge A n1 n1 10
AddEdge A n1 n2 9.2
AddEdge A n1 n2 3.2
AddEdge A n1 n1 1
AddEdge A n1 n3 123
AddEdge A n1 n2 0.1
AddEdge A n1 n2 1.2
AddEdge A n1 n3 3.1415926

ListChildren A n1
ListChildren A n2
ListChildren A n3