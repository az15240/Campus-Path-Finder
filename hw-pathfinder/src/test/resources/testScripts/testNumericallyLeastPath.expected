# Test the numerically sorted output

created graph A
added node n1 to A
added node n2 to A
added node n3 to A
added edge 999.000 from n1 to n3 in A
added edge 10.000 from n1 to n1 in A
added edge 9.200 from n1 to n2 in A
added edge 3.200 from n1 to n2 in A
added edge 1.000 from n1 to n1 in A
added edge 123.000 from n1 to n3 in A
added edge 0.100 from n1 to n2 in A
added edge 1.200 from n1 to n2 in A
added edge 3.142 from n1 to n3 in A

the children of n1 in A are: n1(1.000) n1(10.000) n2(0.100) n2(1.200) n2(3.200) n2(9.200) n3(3.142) n3(123.000) n3(999.000)
the children of n2 in A are:
the children of n3 in A are: